commands.py

```diff
@@ -18,6 +19,11 @@ def want_event(self) -> bool:
         return False
 
 
+class CopilotInsertAsIsCommand(CopilotTextCommand):
```

Comment:
I think this command is no longer required

Reply:
Resolved in f5f0ebd